src-v2/Judge/NTS.Judge/Events/CompetitionChildrenBehind.cs

```diff
@@ -9,18 +9,17 @@ public class CompetitionChildrenBehind : INotBehindParent<Contestant>, INotBehin
     private readonly IRead<Competition> _competitionReader;
     private readonly IParentRepository<Contestant> _contestantParentRepository;
     private readonly IParentRepository<Loop> _loopParentRepository;
-    private Competition? _competition;
+    public Competition? _competition;
```

Comment:
Why public? Fields should always be private. Underscore also should only prefix private fields/properties.